src/libraries/System.Net.Primitives/src/Resources/Strings.resx

```diff
@@ -114,4 +114,28 @@
   <data name="bad_endpoint_string" xml:space="preserve">
     <value>An invalid IPEndPoint was specified.</value>
   </data>
+  <data name="networkerror_unknown" xml:space="preserve">
+    <value>An unknown network error occurred.</value>
+  </data>
+  <data name="networkerror_addressinuse" xml:space="preserve">
+    <value>The requested address is already in use.</value>
```

Comment:
"address" makes sense in terms of `struct sockaddr`, but we should probably use `EndPoint` here as the API has nothing called address. Thoughts?

```suggestion
    <value>The requested EndPoint is already in use.</value>
```